addons/hr_expense/models/hr_expense.py

```diff
@@ -312,3 +304,161 @@ def action_get_attachment_view(self):
         res['domain'] = [('res_model', '=', 'hr.expense'), ('res_id', 'in', self.ids)]
         res['context'] = {'default_res_model': 'hr.expense', 'default_res_id': self.id}
         return res
+
+class HrExpenseSheet(models.Model):
+
+    _name = "hr.expense.sheet"
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _description = "Expense Sheet"
+    _order = "accounting_date desc"
+
+    name = fields.Char(string='Expense Sheet Description', required=True)
+    expense_line_ids = fields.One2many('hr.expense', 'sheet_id', string='Expense Lines', states={'done': [('readonly', True)], 'post': [('readonly', True)]}, copy=False)
+    state = fields.Selection([('draft', 'To Submit'),
+                              ('submit', 'Submitted'),
+                              ('approve', 'Approved'),
+                              ('post', 'Posted'),
+                              ('done', 'Paid'),
+                              ('cancel', 'Refused')
+                              ], string='Status', index=True, readonly=True, track_visibility='onchange', copy=False, default='draft', required=True,
+        help='When the expense sheet request is created the status is \'To Submit\'.\n It is submitted by the employee and request is sent to manager, the status is \'Submitted\'.\
+        \nIf the manager approve it, the status is \'Approved\'.\n If the accountant genrate the accounting entries for the expense sheet request, the status is \'Waiting Payment\'.')
+    employee_id = fields.Many2one('hr.employee', string="Employee", required=True, readonly=True, states={'draft': [('readonly', False)]}, default=lambda self: self.env['hr.employee'].search([('user_id', '=', self.env.uid)], limit=1))
+    partner_id = fields.Many2one('res.partner', related="employee_id.address_home_id", string="Employee Home Address")
+    payment_status = fields.Boolean(string='Payment confirmed', help='The employee greens this button when the payment has been received.')
```

Comment:
The employee manually checks the button when the expense sheet has been reimbursed and when he received the money. After that, the sheet is not displayed by default as it's done and paid.